src/sections/support-page/hero/hero.module.scss

```diff
@@ -0,0 +1,53 @@
+@import '~@bit/bit.base-ui.layout.breakpoints';
+
+.mainGrid {
+	gap: 20px;
+	align-items: center;
+}
+
+.content {
+	margin: 0 auto 25px auto;
+	@media screen and (min-width: $br-md) {
+		margin: 0 0 65px 0;
+	}
+}
+
+.image {
+	max-width: 100%;
```

Comment:
you can use `fullwidth` from alignment component

Reply:
but I dont want it to have `width: 100%`, this is just so it will resize and not overflow off screen in tablet res